raiden_contracts/data/source/raiden/TokenNetworkRegistry.sol

```diff
@@ -32,7 +32,7 @@ contract TokenNetworkRegistry is Utils, Controllable {
     /// @param _secret_registry_address The address of SecretRegistry that's used by all
     /// TokenNetworks created by this contract
     /// that can be chosen at the channel opening
-    /// @param _settle_timeout Number of blocks that need to be mined between a
+    /// @param _settle_timeout Number of seconds that need to elapse between a
```

Comment:
This is the closest I can put the comment: in line `16`, but also in the `constructor` there is the variable `(_)settle_timeout`. The naming is on discussion by the review for the `TokenNetwork` contract. This comment is just a reminder that this naming is used here too. Though you might end up with a naming that makes this just valid as is.